Uplift/Views/GymDetail/GymDetailTodaysClassesCell.swift

```diff
@@ -1,6 +1,6 @@
 //
 //  GymDetailTodaysClassesCell.swift
-//  Fitness
+// Uplift
```

Comment:
could we add a space before all the file headers that were changed? so that they line up with the file name on line 2